data_filter_elasticsearch/cmd/opa-es-filtering/main.go

```diff
@@ -50,6 +50,7 @@ func main() {
 
 	// Index posts.
 	createTestPosts(ctx, client, indexName)
+	fmt.Println("Test post created")
```

Comment:
Is this left from debugging, or it's useful to print this out?

Reply:
Yep just a debug thing, I'll remove it